packages/fiori/src/SideNavigationItem.js

```diff
@@ -129,6 +134,17 @@ class SideNavigationItem extends UI5Element {
 	get _tooltip() {
 		return this.title || this.text;
 	}
+
+	get ariaRoleDescNavigationListItem() {
```

Comment:
Move this to the SideNavigation class as it depends on the "collapsed" property of the whole SideNavigation.